src/GameServer.js

```diff
@@ -385,20 +385,23 @@ GameServer.prototype.mainLoop = function() {
     this.time = local;
 
     if (!this.run) return;
+    
+    // The node & client updating mechanism is perfomance overhauled
+    // Nodes & clients will update periodically and never all on 25ms/50ms
+    // PlayerTracker and all Cell types have own internal update timer which is measured
+    setTimeout(this.updateClients.bind(this), 0);
+    setTimeout(this.moveTick.bind(this), 0);
```

Comment:
Node.js runs in one thread. `setTimeout` method only decreases performance here. I removed all the `setTimeout` methods for my servers.